third_party/move/move-vm/transactional-tests/tests/tracing/simple/abort_msg.masm

```diff
@@ -0,0 +1,13 @@
+//# publish
+module 0x22::test
+
+public fun abort_instr(a: u64)
+    move_loc a
+    ld_const<vector<u8>> [72, 101, 108, 108, 111, 44, 32, 119, 111, 114, 108, 100, 33]
```

Comment:
We can add a test now for a non-utf8 byte array as well?

Reply:
Yeah I had a TODO for that. Done.